docs/setup.md

```diff
@@ -0,0 +1,90 @@
+# Kelvin development
+Kelvin has uses a relatively complicated set of runtime services, so you need to configure a bunch
+of things to start using it.
+
+## Installing Python dependencies
+First, you need to prepare a Python virtual environment and install dependencies of Kelvin, which is
+a Django app. The easiest way of doing that is to use [uv](https://github.com/astral-sh/uv),
+which manages the dependencies of this project. As a first step for working
+with the Python code, install `uv` using some supported approach, for example:
+
+```bash
+$ pip install --user uv==0.4.4
```

Comment:
Current version is `0.4.5`.

Reply:
Sure (I think that I started yesterday in the morning with `0.4.4` and in the evening it was `0.4.5` :laughing:), but we use `0.4.4` everywhere currently, and also in production. If we want to update it, we have to do it across the board (ideally in a separate PR).